### ISSUE TITLE
Feature request: Apple/Metal support for Docker CM workflow

### ISSUE THREAD

**Issue by WarrenSchultz** [mlcommons/cm4mlops#62]
Title says it all :) It would be really helpful if there could be a workflow to use Docker on Macs for simplified testing. In my particular case, ResNet 50 v1.5, but any other models that are functional on Metal would be an added bonus.

**Comment by arjunsuresh** [comment]
Is it okay to do ResNet50 with tensorflow on Metal? If we need pytorch the tricky part is the current reference implementation doesnt support pytorch backend for ResNet50.

**Comment by WarrenSchultz** [comment]
Yeah, we're running it on TF already, so that wouldn't be a change for us, just a more easily repeatable environment.
